docs/source/tutorials/using-your-kit.rst

```diff
@@ -0,0 +1,281 @@
+target speed.
+
+Firstly, how do you smoothly change the robot speed?
+
+It's pretty simple once you understand it:
+
+.. code:: python
+
+    from sr.robot import *
+    import time
+
+    R = Robot()
+
+    for power in range(0, 101)
+          r.motor_board.motors[0].power = power / 100
+          time.sleep(0.01)
+
+
+This code should smoothly speed up your motor from 0 to 1 in 1 second.
+
+The python ``range`` function takes in 2 paramters, ``from`` and ``to``. It then
+simply returns a list of numbers between those two values. It *doesn't* give
+you the last number. (i.e. ``range(0,3)`` will give you a list containing 0, 1, 
+and 2) So if you want the last number you'll need to go one further.
+
+The ``time.sleep`` is there otherwise the code will immediately go to full
+power.
+
+Now try and write some code that:
+- Smoothly starts and stops your robot.
+
+Servos
+------
+
+Servos are a motor which knows what position its at. You can tell it an angle
+and it'll handle turning to that value! 
+
+.. Warning:: Be warened, most servos can't turn a full 360 degrees!
+Always check how far it can move before you design a cool robot arm!
+
+Servos can be set to turn to a specific position. Sadly you can't just tell it
+an angle to turn to in degrees' you can only tell it to go between ``-1`` and 
+``1``. You'll need to measure the angle yourself and work this out if you need
+it!
+
+If you plug a servo in chanel '0' of the servo board, this code will turn it 
+back and forth from minimum to maximum forever:
+
+.. code:: python
+
+    from sbot import *
+    from time import sleep
+
+    r = Robot()
+
+    r.servo_board.servos[0].position = 1
+
+    while True:
+        r.servo_board.servos[0].position = -r.servo_board.servos[0].position
+        sleep(1)
+
+This works because you can get the last position you told the servo to go to 
+with ``blah = r.servo_board.servos[0].position``
+
+Now connect 2 servos to your robot. See if you can spell out
+"Hello" in `Semaphore <https://en.wikipedia.org/wiki/Flag_semaphore>`__.
+You will have to think about which way to orient your servos so they
+can reach all of the positions they need to. You can add paper flags
+to your servos if you want to.
+
+Ultrasound
+----------
+
+An Ultrasound Sensor can be used to measure distances.
+
+The sensor sends a pulse of sound at the object and then measures the time taken
+for the reflection to be heard.
+
+The ultrasound sensors aren't lasers, they have a cone-shaped range, and give 
+you the distance of the nearest large thing. Also ultrasound sensors have both a
+minimum and a maximum range! Make sure you know what the minimum range is for
+your sensor by experimenting with it.
+
+.. code:: python
+
+    from sbot import *
+    from time import sleep
+
+    r = Robot()
+
+    while True:
+        distance = r.arduino.ultrasound_sensors[4, 5].distance()
+        print("Object is " + distance + "m away.")
```

Comment:
Use Python 3 string interpolation syntax.

```suggestion
        print(f"Object is {distance}m away.")
```

Reply:
This was deliberate. Are students going to understand f strings?

Reply:
Let's fix the existing code sample then.

```suggestion
        print("Object is " + str(distance) + "m away.")
```